en-US/D.xml

```diff
@@ -189,7 +189,13 @@
 			<term>dialog box</term>
 			 <listitem>
 				<para>
-					Refer to the Word Usage chapter of the <citetitle>IBM Style Guide</citetitle> for usage information related to this and similar terms.
+					<emphasis>n.</emphasis> Use "dialog box" (<emphasis>not</emphasis> "dialog", "dialogue", "dialogue box", or "window"), to refer to an element that is displayed for a user to interact with a graphical user interface.
+				</para>					
+				<para>
+					Use this term regardless of whether a dialog box is <emphasis>modal</emphasis>, where the user must respond to a prompt before the main content of the interface is enabled again, or is <emphasis>non-modal</emphasis>, where the user can continue interacting with the main content when the dialog box is open.
+				</para>
+				<para>
+					For example: "In the <guibutton>Subscriptions</guibutton> window, click <guibutton>Register</guibutton> to open the <guibutton>Register System</guibutton> dialog box."
 				</para>
```

Comment:
Shouldn't we be using <guilabel> for windows and dialog boxes? Remember that this is not a training course where we use <code> for practically everything, and <guibutton>, etc., for nearly everything else. ^-^

Reply:
Updated.